OpenOffice/python/oxt_metadata/python/main.py

```diff
@@ -199,7 +202,7 @@ def main(*args):
 
     oUDP = Doc.getDocumentProperties().UserDefinedProperties
 
-    if oUDP.getPropertySetInfo().hasPropertyByName("NumberingStyleIndex") == False:
+    if oUDP.getPropertySetInfo().hasPropertyByName("NumberingStyleIndex") is False:
```

Comment:
Please do not use 'is' operator for value comparison even if this works for this particular case.

Reply:
You could use the `not` one. In fact this was a silly one I should have done in the first place...